dts/bindings/flash_controller/flash-controller.yaml

```diff
@@ -0,0 +1,20 @@
+---
+title: flash controller Base Structure
+id: flash-controller
+version: 0.1
+
+description: >
+    This binding gives the base structures for all flash controller devices
+
+properties:
+    label:
+      type: string
+      category: required
+      description: Human readable string describing the device (used by Zephyr for API name)
+      generation: define
+    interrupt-names:
```

Comment:
Why did you feel like interrupt-names, belonged here, but not interrupts or reg?